watchtower-plugin/src/main.rs

```diff
@@ -107,7 +107,7 @@ async fn register(
     );
 
     let tower_net_addr = {
-        if !host.starts_with("http://") {
+        if !host.starts_with("http") {
```

Comment:
let's keep the protocol separator and maybe perform the check as: `!starts_with("http://") && !starts_with("https://")`

as `http` alone might be part of a host name.